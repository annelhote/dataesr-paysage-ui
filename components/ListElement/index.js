import style from './ListElement.module.scss';

export default function ListElement({ children, selectionView }) {
    return (
      <li className={style.Element}>
            {children}
        </li>
    );
}
