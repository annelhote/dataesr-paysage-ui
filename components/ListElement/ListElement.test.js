import React from 'react';
import renderer from 'react-test-renderer';

import ListElement from './index';

it('renders correctly List', () => {
    const tree = renderer.create(<ListElement><p>child</p></ListElement>).toJSON();
    expect(tree).toMatchSnapshot();
});
