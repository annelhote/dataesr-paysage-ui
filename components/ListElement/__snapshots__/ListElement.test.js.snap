// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly List 1`] = `
<li
  className="Element"
>
  <p>
    child
  </p>
</li>
`;
